packages/create-app/templates/default-app/packages/app/package.json.hbs

```diff
@@ -25,6 +25,7 @@
     "@backstage/plugin-permission-react": "^{{version '@backstage/plugin-permission-react'}}",
     "@backstage/plugin-scaffolder": "^{{version '@backstage/plugin-scaffolder'}}",
     "@backstage/plugin-search": "^{{version '@backstage/plugin-search'}}",
+    "@backstage/plugin-search-react": "^{{version '@backstage/plugin-search-react'}}",
```

Comment:
I think just a couple more steps to get this working as expected:

1. You may need to add `@backstage/plugin-search-react` as a peer dependency of `@backstage/create-app`
2. You'll need to update `src/lib/versions.ts` in `@backstage/create-app` to read/set the version number you're using in this template.

^ That should take care of the failing E2E test.

Reply:
Thanks a lot!